csharp/Platform.IO/FileHelpers.cs

```diff
@@ -20,6 +52,22 @@ public static T[] ReadAll<T>(string path)
             return reader.ReadAll<T>();
         }
 
+        /// <summary>
+        /// <para>Reads and returns <typeparamref name="T"/> struct from <paramref name="path"/>.</para>
+        /// <para>Считывает и возвращает структуру типа <typeparamref name="T"/> из <paramref name="path"/>.</para>
+        /// </summary>
+        /// <typeparam name="T">
+        /// <para>The structure type.</para>
+        /// <para>Тип являющийся структурой.</para>
+        /// </typeparam>
+        /// <param name="path">
+        /// <para>The path to the file, from which to read the value of <typeparamref name="T"/> structure.</para>
+        /// <para>Путь к файлу, из которого нужно прочитать значение структуры типа <typeparamref name="T"/>.</para>
+        /// </param>
+        /// <returns>
+        /// <para>A <typeparamref name="T"/> struct value if read from <paramref name="path"/> is successfull, otherwise the default struct value.</para>
```

Comment:
`the default struct value` ⇾ `the default <typeparamref name="T"/> struct value`

Reply:
Why is there `A T struct value `but in the end` The default struct value` ? I mean why are there different articles?